2 Data.sql

```diff
@@ -0,0 +1,27 @@
+use PetDB
+go
+
+delete Pet 
+go
+
+Insert Pet(OwnerName, OwnerAddress, PetType, PetName, Price, Frequency, FirstServiceDate, Active)
```

Comment:
Since you defaulted active to 1 and all customers for now are active, you can take out the active column and they will all default to 1.

Reply:
Fixed